packages/nightingale-structure/src/structure-viewer.ts

```diff
@@ -32,17 +45,47 @@ const viewerOptions = {
   pdbProvider: "pdbe",
   viewportShowControls: PluginConfig.Viewport.ShowControls.defaultValue,
   viewportShowSettings: PluginConfig.Viewport.ShowSettings.defaultValue,
+  extensions: ObjectKeys(Extensions),
 };
 
+const ViewerAutoPreset = StructureRepresentationPresetProvider({
```

Comment:
This is also declared in `molstar/lib/apps/viewer/app.ts` but I assume we should not import from there but rather redeclare this class here https://github.com/molstar/molstar/blob/v3.34.0/src/apps/viewer/app.ts#L484-L510